### DISCUSSION TITLE
personal-development-khordad-1403/

### DISCUSSION THREAD

**Discussion by giscus** [amir-pourmand/hugo-papermod-farsi-template#47] [Announcements]
# personal-development-khordad-1403/

به آخر ماه خرداد رسیدیم و وقت این شده که گزارش توسعه فردی این ماه رو منتشر کنم.
اگر بخوام خیلی خلاصه راجع به خودم و احوالاتم بگم، از نتیجه تصمیم برگشت به یزد خیلی راضیم و واقعیتش اصلاً فکرشو نمی‌کردم انقدر بتونم خوب به کارهام برسم و اهدافم رو پیگیری کنم (و در عین حال ول هم نچرخم).
عجیب‌تر این که کله‌ام خیلی کار می‌کنه و زیاد حرف می‌زنه. انگار تازه داره نفس می‌کشه.

https://aprd.ir/personal-development-khordad-1403/

**Comment by HashemAlavi** [comment]
سلام امیر پورمند عزیز

بابت معرفی ناوال راویکانت و جیمز جانی ممنون؛ یک نگاه انداختم به کانال یوتیوبشون، برام جالب بود و بعداً عمیق‌تر میرم ببینم حرفشون چیه.

در مورد قواعد خوردن گفتی، گفتم شاید برات جالب باشه یکم بیشتر در مورد رژیم مدیترانه‌ای بخونی. از اسمشم معلومه که تو ایران خیلی کسی سراغش نمیره و بیشتر مربوط میشه به جنوب اروپا و سواحل مدیترانه. با اینحال رژیم خیلی عجیب غریبی هم نیست ولی کمک می‌کنه که آدما در بلندمدت با احتمال کمتری دچار مثلث فشار خون، دیابت و بیماری‌های قلبی-عروقی بشن.

اینم برام جالب بود که گروه خونی یکسانی داریم. منم گروه خونیم O+ عه. منتها من تا حالا نرفتم خون بدم، تو ذهنم نبوده که چنین کاری بکنم. با اینحال الآن که گفتی منم به این فکر افتادم که بد نیست برم ببینم خونم رو قبول می‌کنن یا نه.

**Reply by pourmand1376** [reply]
سلام هاشم جان. وقتت بخیر. 

جیمز جانی و ناوال یک تفاوت اساسی دارند و اون اینه که جیمز جانی برای ساختن هر ویدئوش شاید هفته‌ها وقت صرف می‌کنه تا وقت آدم ارزش پیدا بکنه. اما ناوال اصلاً نیازی به ادیت نداره و به قول امین آرامش کمتر کسی رو پیدا می‌کنید که بتونه آنقدر حرف‌های مفید در یک مصاحبه ساده بزنه. 

کامنتت رو گذاشتم کنار که ببینم رژیم مدیترانه‌ای چیه و بعد جواب بدم. به نظر چیز جالبی میاد. باید یه حالت ترکیبی ای برای خودم پیدا کنم. قطعاً با کاهش شکر نمی‌تونم مخالفت کنم اما لبنیات بخش مهمی از زندگیم هست و کم‌کردنش برام سخته. 

ولی می‌تونم توصیه‌های کلی‌اش رو در نظر بگیرم و امیدوارم این جنس وسط بازی کلیت رژیم رو زیر سوال نبره :) 

راجع به انتقال خون هم من سال‌هاست که میرم میدم. یه جورایی احساسم اینه که دین بر گردنم هست؛ چون خانم‌ها که معمولاً به سختی می‌تونند خون بدن و بخش بزرگی از آقایون هم خون‌شون به دلایل مختلف به درد بیمار نمی‌خوره. اینه که یه جورایی وظیفه عمومی می‌دونم.

پی‌نوشت: من گزارش ماهانه توسعه فردی‌ات رو هم مطالعه می‌کنم و اتفاقاً قبل از دیدن کامنتت، خونده بودم. خیلی ساده و زیبا می‌نویسی. برام جالب بود که به درستی به دنبال منابع دیگه هم برای یادگیری هوش مصنوعی رفتی. من نهایتاً هر دو هفته‌ای یک قسمت پادکست بدم که شاید بعد یک سال یک چیز دندان گیری از توش دربیاد. خصوصاً این که هر چی جلوتر میرم متوجه میشم قسمت‌های قبل چقدر ضعیف‌ بوده و بعضی وقت‌ها تصمیم می‌گیرم حذف‌شون کنم یا از نو ضبط کنم.

**Reply by HashemAlavi** [reply]
در مورد رژیم مدیترانه‌ای منم هنوز کامل نرفتم سمتش. حس می‌کنم یکم وقت گیر هم هست و باید سر حوصله بری خرید کنی، سر حوصله غذا بپزی و یکمی هم از رژیم‌های معمولی ما گرون‌تر در میاد. ولی ارزشش رو داره که سهم بیشتری بهش بدیم.

خیلی خوبه که انقدر به اهدای خون پایبندی. منم این دو سه هفته که امتحاناتم تموم بشه حتماً میرم ببینم شرایطش رو دارم یا نه. نمی‌دونم چرا تا حالا جدی نگرفته بودم این موضوع رو.

در مورد پادکست: اول اینکه خیلی ممنون که وقت می‌ذاری و محتوای قابل فهم برای ما می‌سازی. مخصوصاً که فکر نکنم تو فضای فارسی چنین پادکستی باشه (اینی که گفتم مطمئن نیستم ازش). راستش نمی‌خوام تعریف بیخود کنم و بگم هیچ ایرادی نداره و عالیه و از این تعارف‌های الکی. ولی من تا الآن با اپیزودهایی که منتشر کردی اوکی بودم. بنظرم توضیحاتت روان بود، کیفیت صدا هم خوب بود و محتوا رو هم دوست داشتم.
